tensorflow-ops/tests/OpsTest.hs

```diff
@@ -104,6 +104,20 @@ testRereadRef = testCase "testReRunAssign" $ TF.runSession $ do
     f1 <- TF.run w
     liftIO $ (0.0, 0.1) @=? (TF.unScalar f0, TF.unScalar f1)
 
+-- | Test einstein summation
```

Comment:
Thank you for adding a test!

Capitalize Einstein? Use . to finish the sentence.

Reply:
This has been addressed.